client/src/modules/ipr_tax/templates/rate.tmpl.html

```diff
@@ -0,0 +1,3 @@
+<div class="ui-grid-cell-contents text-right">
+  {{ row.entity.rate }} %
```

Comment:
You could actually make a filter for this.  [See this for example](https://gist.github.com/jeffjohnson9046/9470800).